linode/manager
# refactor: [M3-6494] - React Query for Linode Migrate Dialog

## Description 📝

- Use React Query hooks in `<MigrateLinode />`

## Major Changes 🔄
- Remove some usages of `listToItemsByID` 🎉
- Removes some unnecessary error states that were not used (`regionError`, `acceptError`) 🗑️
- Use all React Query hooks in `<MigrateLinode />`

## Preview 📷

> **Note**: No behavior or UI should change

![Screenshot 2023-05-30 at 4 59 05 PM](https://github.com/linode/manager/assets/115251059/77367644-6581-44c8-82f5-bacf6bcce42d)

## How to test 🧪
1. Spin up a Linode of any kind
2. Use the action menu on the Linode Details page or Linodes Landing page to open the **`Migrate`** dialog
3. Migrate your linode to a new region
4. Verify the migration successfully happens
5. Repeat the steps above and try breaking the Migrate dialog
